.eslintignore

```diff
@@ -1,3 +1,4 @@
 /**/node_modules/
 /**/libs/
 /**/_next/
+!.storybook/
```

Comment:
eslint ignores things starting with `.` by default, but we don't want it to for things in the `.storybook` dir, so prefixing with a `!` means 'don't ignore this'